src/app/customer/customer-detail/customer-detail.component.spec.ts

```
@@ -1,32 +1,163 @@
+          } as Item,
+        ],
+      }),
+    );
   });
 
   beforeEach(() => {
     fixture = TestBed.createComponent(CustomerDetailComponent);
     component = fixture.componentInstance;
+    loader = TestbedHarnessEnvironment.loader(fixture);
     fixture.detectChanges();
   });
```

Biome

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)